static/app/utils/theme.tsx

```diff
@@ -50,7 +50,7 @@ const lightColors = {
   pink100: 'rgba(250, 51, 150, 0.1)',
 };
 
-const darkColors = {
+export const darkColors = {
```

Comment:
@billyvg Think you had some concerns about exporting raw, un-aliased colors from `theme.tsx`?

Reply:
@vuluongj20 Yeah I think we want to avoid the situation where people import and use them directly.

This might be ok because:
1) you're only using this for storybook
2) it's not exposed via our theme

Here are some additional things we could do - 

Good:
Add a comment for both of these exports to warn 

Better:
Add a lint rule to disable importing `lightColors` and `darkColors`  from `theme.tsx` (this should be relatively easy to do with https://eslint.org/docs/rules/no-restricted-imports)

Best:
Add a lint rule to disable importing from `theme.tsx` at all (this is difficult since we import from `theme.tsx` a lot, and out of scope for this PR)

Reply:
Let me know if you are interested in adding a lint rule, I can help you if needed.

Reply:
Added some comments, and created a PR with import lint rules: https://github.com/getsentry/eslint-config-sentry/pull/119